.github/workflows/test.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           - python: "3.8"
             db: postgres
           - python: "3.8"
-            nbclassic: nbclassic
+            legacy_notebook: legacy_notebook
```

Comment:
`nbclassic` is a bit confusing to call the matrix entry where we explicitly do _not_ use the package `nbclassic`, it's the legacy package `notebook`.